src/Build.UnitTests/MockTask.cs

```diff
@@ -425,17 +425,7 @@ internal sealed class MyTaskItem : ITaskItem
     {
         #region ITaskItem Members
 
-        public string ItemSpec
-        {
-            get
-            {
-                return "foo";
-            }
-            set
-            {
-                // do nothing
-            }
-        }
+        public string ItemSpec { get; set; }
```

Comment:
this property was previously initialized to '"foo"', is it now uninitialized by design?

Reply:
It was a dummy mock implementation that was actually never used - so I could go fwd without risk of breaking anything (it's only used in our tests). More real-like behavior was needed for new detailed tests